.github/scripts/release.py

```diff
@@ -66,19 +66,20 @@ def get_local_package_index_json():
     return data
 
 def get_platform_data_struct_copy(pkg_index):
-    return copy.deepcopy(pkg_index['packages'][0]['platforms'][0])
+    return copy.deepcopy(pkg_index['packages'][0]['platforms'])
```

Comment:
Any modification here in behavior or feature added in this release.py script? Can you explain the motivation behind this changes?

Reply:
![image](https://github.com/Infineon/XMC-for-Arduino/assets/51888075/82dd8f95-bdb5-49f3-8788-15f81df90cc5)
Here I checked my commit note. This script will take the local example as template, so we can easily change help url/ compiler version and so on.

Reply:
All right!